src/rred_reports/reports/schools.py

```diff
@@ -186,7 +186,7 @@ def get_outcome_from_summary(school_df: pd.DataFrame, outcome_type: str) -> int:
             "po_discontinued": get_outcome_from_summary(filtered_summary_table, "Discontinued"),
             "po_referred_to_school": get_outcome_from_summary(filtered_summary_table, "Referred to school"),
             "po_incomplete": get_outcome_from_summary(filtered_summary_table, "Incomplete"),
-            "po_left_school": get_outcome_from_summary(filtered_summary_table, "Left school"),
+            "po_left_school": get_outcome_from_summary(filtered_summary_table, "Left School"),

```

Comment:
would it be worth making the `outcome_type` in `get_outcome_from_summary` case insensitive? or does `exit_outcome` in the `outcome_df` come from our processing making that not necessary

Reply:
mmm they want them displayed as they're entered, for the further tables, but we could make a copy just for the summary table and then make them all lowercase